render_delegate/light.cpp

```diff
@@ -641,7 +642,6 @@ class HdArnoldSimpleLight : public HdLight {
     HdDirtyBits GetInitialDirtyBitsMask() const override;
 
 private:
-    HdArnoldRenderDelegate* _delegate; ///< Pointer to the Render Delegate.
```

Comment:
Let's just get rid of simple light, we don't need it anymore. This is just a leftover from older times, nowadays scene delegates should supply us with correct, UsdLux light types.